sdk/eventhub/event-hubs/samples-dev/sendBufferedEvents.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT Licence.
+
+/**
+ * @summary Demonstrates how to send events to an Event Hub from a buffered producer.
+ *
+ * @azsdk-weight 60
+ */
+
+import { EventHubBufferedProducerClient, OnSendEventsErrorContext } from "@azure/event-hubs";
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const connectionString = process.env["EVENTHUB_CONNECTION_STRING"] || "";
+
+async function handleError(ctx: OnSendEventsErrorContext): Promise<void> {
+  console.log(JSON.stringify(ctx.error), undefined, "  ");
+}
+
+export async function main(): Promise<void> {
+  console.log(`Running sendEvents sample`);
+
+  const producer = new EventHubBufferedProducerClient(connectionString, {
```

Comment:
It would be nice to demonstrate how to configure the count/timeout options here

Reply:
done!